### PR TITLE
Devadv 1159 jmunta

### PR DESCRIPTION
**Pull request by jdmuntacb**
Changes with gitpod and couchbase 7.0 latest official docker image seem ok.
See the below screenshot.

Screen Shot 2021-08-14 at 10 06 28 AM

### CODE REVIEW

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -1,5 +1,20 @@
-FROM deniswsrosa/couchbase7-5868-gitpod
+FROM couchbase:latest
```
Comment (nithishr) [inline .gitpod.Dockerfile:1]:
```suggestion
FROM couchbase:7.0.0
```
It would be safer to add the image version for future compatibility issues

**Review by nithishr — CHANGES_REQUESTED** [review]
I also noticed a couple of improvements not directly related to this PR. 
* Can we move the log usage task above the app running task so that we get the app logs when the app is running instead of the log usage terminal?
* The tests were failing on gitpod & also the search was causing an internal server error.

File: .gitpod.Dockerfile (hunk 1 above)
Comment (jdmuntacb) [reply .gitpod.Dockerfile:1]: @nithishr On the version wise, we can keep the latest in similar to other examples that I had merged. Later, we can think as to the specific as needed. Hope this is ok as discussed.

Let me see the other suggestions that you had mentioned.

**Comment by jdmuntacb** [comment]
@nithishr 
Now the tests passed on the gitpod as well as locally. There may be some changes later that can be done for the encrypted password verification. 
Also, the search is working fine.
See the below screenshots. Please verify so that we can merge this change.

Screen Shot 2021-08-23 at 8 14 15 PM

Screen Shot 2021-08-23 at 8 42 10 PM

**Review by nithishr — CHANGES_REQUESTED** [review]
The test pass & the outputs look fine. 

Just one small comment: Can we move the logging task in gitpod above the app task in `.gitpod.yml` so that the users get the application logs instead of logging inside the gitpod environment?

**Comment by jdmuntacb** [comment]
@nithishr Ok. Move the log use task before app task. Could you please verify?

**Review by nithishr — APPROVED** [review]

**Pull request merged**